db/migrate/20170703182119_create_suggestions_job_statuses.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSuggestionsJobStatuses < ActiveRecord::Migration[5.1]
+  def change
+    create_table :suggestions_job_statuses do |t|
+      enable_extension "hstore"
+      t.string "message"
+      t.hstore "payload"
+      t.boolean "done", default: false, null: false
+      t.string "job_id"
```

Comment:
I have communication concerns about DB columns that end in `_id` in a Rails app -- to me they make me think that the `Job` AR model has a `has_many :suggestions_job_statuses` relation.

Do you mind renaming this to `jobid` or `job_identifier` instead?